src/Humanizer.Tests/FrequencyHumanizeTest.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Xunit;
+using Xunit.Extensions;
+using System;
+
+namespace Humanizer.Tests
+{
+    public class FrequencyHumanizeTest
+    {
+
+        [Fact]
+        public void EveryTwelveHours(TimeSpan timespan, string expectedSentence)
+        {
+            Assert.True(Frequency.Every(new TimeSpan(12, 0, 0)).Humanize() == "roughly every twelve hours");
+        }
+
+        [Fact]
+        public void EveryThreeDaysAndAHalf(TimeSpan timespan, string expectedSentence)
```

Comment:
I am no expert in English but I think this should read as 'roughly every three and a half days'